service/src/main/java/com/fecklessweasel/service/UserServlet.java

```diff
@@ -0,0 +1,84 @@
+package com.fecklessweasel.service;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import com.fecklessweasel.service.datatier.SQLSource;
+import com.fecklessweasel.service.datatier.SQLInteractionInterface;
+import com.fecklessweasel.service.objectmodel.ServiceException;
+import com.fecklessweasel.service.objectmodel.ServiceStatus;
+import com.fecklessweasel.service.objectmodel.User;
+import com.fecklessweasel.service.objectmodel.UserSession;
+
+/**
+ * A servlet that handles post requests from an HTML form and produces
+ * new users.
+ * @author Christian Gunderman
+ */
+@WebServlet("/servlet/user")
```

Comment:
Servlet name is /servlet/user in keeping with patterns established in backend. Rather than RPC's that can be called I like to think of Servlets as resources that can be modified. I called this the user servlet because we could use it to delete users with a hidden input field. That's how I do login/logout.